src/renderers/shared/stack/event/EventConstants.js

```diff
@@ -18,7 +18,7 @@ var PropagationPhases = keyMirror({bubbled: null, captured: null});
 /**
  * Types of raw signals from the browser caught at the top level.
  */
-var topLevelTypes = keyMirror({
+var topLevelTypes = {
```

Comment:
ReactTestUtils is doing a for loop on this, hence why we need to keep it as an object and can't just make it just live in the type system:

https://github.com/facebook/react/blob/6a659606415d4853026df6e6d525274fdc7d35ea/src/test/ReactTestUtils.js#L619-L629